presto-docs/src/main/sphinx/connector/hive-caching.rst

```diff
@@ -190,10 +190,3 @@ The following example query returns the average cache hit ratio for the ``hive``
   FROM jmx.current."rubix:catalog=hive,name=stats"
   WHERE NOT is_nan(cache_hit);
 
-Limitations
------------
-
-Caching does not support user impersonation. It does not take any user-specific
```

Comment:
THis is not supper accurate as we are only supporting impersonation in kerberos authentication mode (if `hive.hdfs.authentication.type=KERBEROS`).
We still do not support impersonation if `hive.hdfs.authentication.type=NONE`.

cc: @sopel39 @findepi